endpoints/citrus-http/src/main/java/org/citrusframework/http/client/HttpResponseErrorHandler.java

```diff
@@ -46,7 +46,7 @@ protected void handleError(ClientHttpResponse response, HttpStatusCode statusCod
         if (errorHandlingStrategy.equals(ErrorHandlingStrategy.PROPAGATE)) {
             throw new HttpErrorPropagatingException(response.getStatusCode(), response.getStatusText(), response.getHeaders(), getResponseBody(response), getCharset(response));
         } else if (errorHandlingStrategy.equals(ErrorHandlingStrategy.THROWS_EXCEPTION)) {
-            super.handleError(response);
+            super.handleError(response, statusCode, url, method);
```

Comment:
@bbortt This fix should be added to Citrus Main

Reply:
upstream `cherry-pick`: https://github.com/citrusframework/citrus/pull/1279.